tools/cross_compile.go

```diff
@@ -0,0 +1,69 @@
+// +build none
+
+package main
```

Comment:
Please add comment above:

// cross_compile builds ditaa for all known platforms. Call with: go run tools/cross_compile.go